js/noms/package.json

```diff
@@ -42,7 +41,7 @@
   "scripts": {
     "pretest": "eslint src/ && flow src/",
     "ts": "mocha --ui tdd --timeout=4000 --reporter spec --slow=50 --invert --grep='LONG:' --compilers js:babel-core/register src/*-test.js",
-    "test": "mocha --ui tdd --timeout=6000 --reporter dot --compilers js:babel-core/register src/*-test.js",
+    "test": "mocha --ui tdd --timeout=10000 --reporter dot --compilers js:babel-core/register src/*-test.js",
```

Comment:
This is unrelated to this change but the slow tests were timing out on my machine (it was super slow due to Apple update :cry:)